js/main.js

```diff
@@ -303,24 +310,13 @@ function updateLegendDataSource(dataSource) {
     $("#legend #dataSource").html('<a href="' + url + '">' + title + '</a>');
 }
 
-
-/*
- * Convert a city ID to a UI label.
- */
-function cityIdToLabel(s) {
-    return s.replace(/(^|\s|\-)([a-zA-ZäöüÄÖÜß])/g, function (x) {
-        return x.toUpperCase();
-    });
-}
-
-
 /*
  * Set the current city.
  *
  * `city` is the city ID.
  */
 function setCity(city) {
-    var filename = 'cities/' + city + '.json';
+    var filename = 'cities/data/' + city + '.json';
```

Comment:
I don't think that there should be a problem with keeping them all in the same directory: The only file that we explicitly refer to will be `cities.json`, all other filenames will be listed in there (or inferred from IDs listed in there). So there is no need for directory listings and no source of confusion.